support-files/mariadb.service.in

```diff
@@ -48,10 +48,13 @@ User=mysql
 Group=mysql
 
 # CAP_IPC_LOCK To allow memlock to be used as non-root user
+# These are enabled by default
+AmbientCapabilities=CAP_IPC_LOCK
+
 # CAP_DAC_OVERRIDE To allow auth_pam_tool (which is SUID root) to read /etc/shadow when it's chmod 0
 #   does nothing for non-root, not needed if /etc/shadow is u+r
 # CAP_AUDIT_WRITE auth_pam_tool needs it on Debian for whatever reason
-AmbientCapabilities=CAP_IPC_LOCK CAP_DAC_OVERRIDE CAP_AUDIT_WRITE
+CapabilityBoundingSet=CAP_DAC_OVERRIDE CAP_AUDIT_WRITE
```

Comment:
Don't this is needed .. I changed mine to only have 

`AmbientCapabilities=CAP_IPC_LOCK` and this seems to work

Reply:
per above comments in file these are only for the pam integration of MariaDB.

Reply:
> During an [execve(2)](https://www.man7.org/linux/man-pages/man2/execve.2.html), the capability bounding set is ANDed with the file permitted capability set

what is the "file permitted capability set" for auth_pam_tool ?

Reply:
@vuvova as it doesn't seem to be defined anywhere explicitly, it defaults to "everything". Then this line limits it to CAP_DAC_OVERRIDE and CAP_AUDIT_WRITE.